src/templates/en/2020/methodology.html

```diff
@@ -0,0 +1,426 @@
+      </p>
+
+      <p>
+        Some metrics are completely out of reach. For example, we don't necessarily have the ability to detect the tools used to build a website. If a website is built using create-react-app, we could tell that it uses the React framework, but not necessarily that a particular build tool is used. Unless these tools leave detectible fingerprints in the website's code, we're unable to measure their usage.
+      </p>
+
+      <p>
+        Other metrics may not necessarily be impossible to measure but are challenging or unreliable. For example, aspects of web design are inherently visual and may be difficult to quantify, like whether a page has an intrusive modal dialog.
+      </p>
+    </section>
+
+    <section>
+      <h2 id="tools"><a href="#tools" class="anchor-link">Tools</a></h2>
+
+      <p>
+        The Web Almanac is made possible with the help of the following open source tools.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="webpagetest"><a href="#webpagetest" class="anchor-link">WebPageTest</a></h3>
+
+      <p>
+        <a href="https://www.webpagetest.org/">WebPageTest</a> is a prominent web performance testing tool and the backbone of HTTP Archive. We use a <a href="https://github.com/WPO-Foundation/webpagetest-docs/blob/master/user/Private%20Instances/README.md">private instance</a> of WebPageTest with private test agents, which are the actual browsers that test each web page. Desktop and mobile websites are tested under different configurations:
+      </p>
+      <div class="table-wrap"><div class="table-wrap-container">
+      <table>
+        <thead>
+          <tr>
+              <th>Config</th>
+              <th>Desktop</th>
+              <th>Mobile</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Device</td>
+            <td>Linux VM</td>
+            <td>Emulated Moto G4</td>
+          </tr>
+
+          <tr>
+            <td>User Agent</td>
+            <td>
+              Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36 PTST/200805.230825
+            </td>
+            <td>
+              Mozilla/5.0 (Linux; Android 6.0.1; Moto G (4) Build/MPJ24.139-64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.146 Mobile Safari/537.36 PTST/200815.130813
+            </td>
+          </tr>
+
+          <tr>
+            <td>Location</td>
+            <td>
+              Redwood City, California, USA<br>
+              The Dalles, Oregon, USA
+            </td>
+            <td>
+              Redwood City, California, USA<br>
+              The Dalles, Oregon, USA
+            </td>
+          </tr>
+
+          <tr>
+            <td>Connection</td>
+            <td>Cable (5/1 Mbps 28ms RTT)</td>
+            <td>3G (1.600/0.768 Mbps 300ms RTT)</td>
```

Comment:
Testing is at 4G speeds for Mobile (9Mbps, 170ms RTT)

Reply:
3G Fast for Lighthouse (which is the Lighthouse default) (1.6Mbps, 150Ms RTT)

Reply:
@pmeenan looking at an [example run for 2020](https://webpagetest.httparchive.org/result/200801_Mx40_7VHJ/) it says it's 3G

```
from: "Redwood City, CA - <b>Chrome</b> - <b>Emulated Motorola G (gen 4)</b> - <b>3G</b>",
connectivity: "3G",
bwDown: 1600,
bwUp: 768,
latency: 300,
```

Which matches what we originally had. So you sure about this change?

Reply:
I stand corrected.  The [code](https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/bulktest/batch_lib.inc#L94) is supposed to swap it out for 4G but I think Mobile uses "California2" so it never worked.  3G is correct after all.